includes/header.php

```diff
@@ -194,6 +194,8 @@ function gtag() {
         <div class="container">
           <?php
           if (isset($md_github_url) and $md_github_url) {
+              $md_github_url = str_replace('contributing', 'developers', $md_github_url);
+              $md_github_url = str_replace('docs', '', $md_github_url);
```

Comment:
```suggestion
              $md_github_url = str_replace('/contributing/', '/developers/', $md_github_url);
              $md_github_url = str_replace('/docs', '', $md_github_url);
```

Reply:
I think this should avoid off-target effects.

Reply:
Ah, I always forget that you can just click "commit suggestion" 🙄